### PR TITLE
Fix email prefill

### PR DESCRIPTION
**Pull request by danielmolnar**
This PR fixes a bug described in issue #243. 

https://user-images.githubusercontent.com/77627178/234714984-f73b7e2d-e8b4-4c47-b4ab-e78de7c56080.mov

### CODE REVIEW

File: lib/src/core/widgets/backdrop/step_page_scaffold.dart [hunk 1]
```diff
@@ -288,7 +288,7 @@ class StepPageScaffoldState extends State<StepPageScaffold> {
   }
 }
 
-enum StepPageAlignemnt {
+enum StepPageAlignment {
```
Comment (passsy) [inline lib/src/core/widgets/backdrop/step_page_scaffold.dart:291]: lol

File: lib/src/feedback/steps/step_3_screenshot_overview.dart [hunk 2]
```diff
@@ -77,7 +82,10 @@ class Step3NotAttachments extends StatelessWidget {
                         label: context
                             .l10n.feedbackStep3ScreenshotOverviewSkipButton,
                         trailingIcon: Wirecons.chevron_double_right,
-                        onTap: context.feedbackModel.goToNextStep,
+                        onTap: () async {
+                          if (!mounted) return;
```
Comment (passsy) [inline lib/src/feedback/steps/step_3_screenshot_overview.dart:86]: you can leave it a StatelessWidget and use `context.mounted`

File: test/feedback_flow_test.dart [hunk 3]
```diff
@@ -129,6 +129,63 @@ void main() {
       expect(submittedFeedback.message, 'feedback with labels');
     });
 
+    testWidgets(
+        'E-Mail is prefilled when user email is set and skips screenshot',
```
Comment (passsy) [inline test/feedback_flow_test.dart:133]: We also need a test: "user adjusts prefilled email and it doesn't get reset when returning to email step"

Comment (danielmolnar) [reply test/feedback_flow_test.dart:133]: Good call!

File: lib/src/feedback/steps/step_3_screenshot_overview.dart (hunk 2 above)
Comment (danielmolnar) [reply lib/src/feedback/steps/step_3_screenshot_overview.dart:86]: Done [b236875](https://github.com/wiredashio/wiredash-sdk/pull/272/commits/b236875d2a688e066970e5b7a72592c8feaf66ec)

File: test/util/robot.dart [hunk 4]
```diff
@@ -219,9 +219,9 @@ class WiredashTestRobot {
     await tester.tap(
       step.text('l10n.feedbackStep3ScreenshotOverviewSkipButton').finder,
     );
-    await tester.pumpAndSettle();
-    await tester.pumpAndSettle();
+    await tester.pumpAndSettle(const Duration(seconds: 1));
```
Comment (passsy) [inline test/util/robot.dart]: this duration is specific to the test, move it there.

File: test/feedback_flow_test.dart (hunk 3 above)
Comment (danielmolnar) [reply test/feedback_flow_test.dart:133]: Done [78a1c4d](https://github.com/wiredashio/wiredash-sdk/pull/272/commits/78a1c4dbe0968c91ebd7f1d0582cecb8196adfea)

File: test/util/robot.dart (hunk 4 above)
Comment (danielmolnar) [reply test/util/robot.dart]: Done [c171089](https://github.com/wiredashio/wiredash-sdk/pull/272/commits/c171089f408a3a9cc1b7c7b76c6ff7534c89b968)

File: test/util/robot.dart [hunk 5]
```diff
@@ -286,13 +286,13 @@ class WiredashTestRobot {
 
   Future<void> goToPrevStep() async {
     final oldStatus = services.feedbackModel.feedbackFlowStatus;
+    final texts = spotTexts('l10n.feedbackBackButton');
     final backdropStatus = services.backdropController.backdropStatus;
-    final backbutton = find.text('l10n.feedbackBackButton');
 
     if (backdropStatus == WiredashBackdropStatus.centered) {
-      await tester.tap(backbutton.last);
+      await _tap(texts.last());
```
Comment (danielmolnar) [inline test/util/robot.dart:293]: @passsy This is the change I had to make for the test to work again after the latest stable release.

**Comment by passsy** [comment]
Thanks so much for your contribution ❤️

**Pull request merged**